.changes/next-release/bugfix-S3TransferManager-f1c2208.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "S3 Transfer Manager", 
+    "contributor": "", 
+    "type": "bugfix", 
+    "description": "Fixed issue where errors were being wrapped by SdkClientException"
```

Comment:
Can we update the title of this PR , currently it has the username in it.
Also can we be specific as in "Fixed issues in `resumeDownloadFile` API  where errors were being wrapped by SdkClientException"

Reply:
Updated title and description